### ISSUE TITLE
Start saving observations in ensembles that use observations

### ISSUE THREAD

**Issue by oyvindeide** [equinor/ert#3487]
We would like to start internalizing the observations, at least for experiments that rely on them, namely ensemble_smoother, iterated_ensemble_smoother and es_mda. So we should start saving them in: https://github.com/equinor/ert/blob/69e0d45ca6e5caa776a479643b215d0e4e44aa6d/ert_shared/models/ensemble_smoother.py#L34

**Comment by dafeda** [comment]
What is this blocked by?

**Comment by pinkwah** [comment]
Blocked by lack of database or database adjacent functionality.

**Comment by oyvindeide** [comment]
Think we will close this for now as it is not part of the current scope.
